lib/sinon/stub.js

```diff
@@ -18,6 +18,14 @@ function stub(object, property) {
         throw new TypeError("stub(obj, 'meth', fn) has been removed, see documentation");
     }
 
+    if (
```

Comment:
It would be great to move this into a function, so it can be re-used (upcoming `sinon@next` has a new `replace` method).